apps/gustaf/create_stellarmatches_from_file.h

```diff
@@ -81,157 +93,170 @@ void _getShortIds(StringSet<TId> & shortIds, StringSet<TId> & longIds)
+		TAlign localAlign;
+		resize(rows(localAlign), 2);
+		setSource(row(localAlign, 0), host(dbInf));
+		setSource(row(localAlign, 1), host(queryInf));
+		TRow & row1 = row(localAlign, 0);
+		TRow & row2 = row(localAlign, 1);
 
 
-        // Set begin and end positions of align
-        setBeginPosition(row2, lmStore.matchStore[i].queryBeginPos);
-        setBeginPosition(row1, lmStore.matchStore[i].subjectBeginPos);
+		// Set begin and end positions of align
+		setBeginPosition(row2, (*it).queryBeginPos);
+		setBeginPosition(row1, (*it).subjectBeginPos);
 
-        // setBeginPosition(row1, 0);
-        // setBeginPosition(row2, 0);
-        setEndPosition(row1, lmStore.matchStore[i].subjectEndPos);
-        setEndPosition(row2, lmStore.matchStore[i].queryEndPos);
+		// setBeginPosition(row1, 0);
+		// setBeginPosition(row2, 0);
+		setEndPosition(row1, (*it).subjectEndPos);
+		setEndPosition(row2, (*it).queryEndPos);
 
-        unsigned gapIndex = 0;
-        // Inserting gaps into rows according to cigar line
-        if (length(lmStore.cigarStore) > lmStore.matchStore[i].id)
-        {
-            String<CigarElement<> > const & cigar = lmStore.cigarStore[lmStore.matchStore[i].id];
-            for (unsigned j = 0; j < length(cigar); ++j)
-            {
-                // std::cout << cigar[j].count << cigar[j].operation;
-                if (cigar[j].operation == 'I')
-                {
-                    for (unsigned gap = 0; gap < cigar[j].count; ++gap)
-                    {
-                        insertGap(row1, gapIndex);
-                        ++gapIndex;
-                    }
-                }
-                else if (cigar[j].operation == 'D')
-                {
-                    for (unsigned gap = 0; gap < cigar[j].count; ++gap)
-                    {
-                        insertGap(row2, gapIndex);
-                        ++gapIndex;
-                    }
-                }
-                else
-                    gapIndex += cigar[j].count;
-            }
-        }
-        // Create Stellar match and append it to stQueryMatches
-        StellarMatch<TSequence, TId> match(localAlign, databaseIds[iDB], orientation);
-        appendValue(stQueryMatches[iQuery].matches, match);
+		unsigned gapIndex = 0;
+		// Inserting gaps into rows according to cigar line
+		if (length(lmStore.cigarStore) > (*it).id)
+		{
+		    String<CigarElement<> > const & cigar = lmStore.cigarStore[(*it).id];
+		    for (unsigned j = 0; j < length(cigar); ++j)
+		    {
+			// std::cout << cigar[j].count << cigar[j].operation;
+			if (cigar[j].operation == 'I')
+			{
+			    for (unsigned gap = 0; gap < cigar[j].count; ++gap)
+			    {
+				insertGap(row1, gapIndex);
+				++gapIndex;
+			    }
+			}
+			else if (cigar[j].operation == 'D')
+			{
+			    for (unsigned gap = 0; gap < cigar[j].count; ++gap)
+			    {
+				insertGap(row2, gapIndex);
+				++gapIndex;
+			    }
+			}
+			else
+			    gapIndex += cigar[j].count;
```

Comment:
For easier reading you could also add curly brackets to the last else. 
In general you can leave the brackets if every statement of the if-else if-else block consists of only one line. 
As soon as there is one block with multiple lines you should add the curly brackets to all blocks.